native/packages/react-native-bpk-component-banner-alert/stories.js

```diff
@@ -0,0 +1,259 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react-native';
+import { StyleSheet, View, Platform } from 'react-native';
+import BpkText from 'react-native-bpk-component-text';
+import BpkBannerAlert, { ALERT_TYPES } from './index';
+
+const tokens = Platform.select({
+  ios: () => require('bpk-tokens/tokens/ios/base.react.native.common.js'), // eslint-disable-line global-require
+  android: () => require('bpk-tokens/tokens/android/base.react.native.common.js'), // eslint-disable-line global-require
+})();
+
+const {
+  colorGray500,
+  spacingSm,
+  spacingBase,
+} = tokens;
+
+const styles = StyleSheet.create({
```

Comment:
There is a new standardised layout for the storybooks, can you double check with @shaundon on how to use it?

Reply:
Sure - have done :)